data/src/main/java/com/microsoft/azure/kusto/data/Utils.java

```diff
@@ -34,6 +34,8 @@ static Results post(String url, String payload, InputStream stream, Integer time
         } else {
             httpClient = HttpClients.createSystem();
         }
+
+        url = url.replace(" ", "%20");
```

Comment:
Please don't do this manually, check this out :
https://stackoverflow.com/a/10786112/929164